crates/project/src/lsp_command.rs

```diff
@@ -181,6 +181,11 @@ pub(crate) struct LinkedEditingRange {
     pub position: Anchor,
 }
 
+#[derive(Debug)]
+pub(crate) struct GetDocumentDiagnostics {
```

Comment:
Is implementing such a command via `LspCommand` right? The current method for getting diagnostics is done directly in the `LspStore`, but I noticed there is some work for moving code to support SSH. So, this is an open question.